api/client/webclient/webclient.go

```diff
@@ -54,6 +54,13 @@ const (
 	// based on the specified group. e.g. some groups might need to update
 	// before others.
 	AgentUpdateGroupParameter = "group"
+
+	// AgentUpdateHostParameter is the parameter used to specify the updater
+	// host during a Ping() or Find() query.
+	// The proxy server will modulate the auto_update part of the PingResponse
+	// based on the specified host ID. e.g. canary hosts might need to update
+	// before others.
+	AgentUpdateHostParameter = "host"
```

Comment:
I read the RFD again and while we do specify the `host` and `group` parameters, I think `host` was initially intended to be the hostID before we chose to create a new updater UUID.

The RFD has a mix of
- update_uuid / update_group
- host / group

I think this can get confusing as is some places we are also dealing with real host IDs, which are different from the updater one. WDYT about renaming the `host` field? The rest of the RFD is talking about updater UUID, so maybe `updaterID` or `updaterUUID` would fit?

Reply:
I'll change it to `UpdaterID` to match teleport-update.

Reply:
Ended up going with `UpdateID` for consistency with `UpdateGroup`. Let me know if that looks good.

Reply:
That's perfect, thank you